### ISSUE TITLE
Graph coloring not working in Windows 10

### ISSUE THREAD

**Issue by Xtopher98** [dracula/gitkraken#7]
Use of `graph-color` does not apply correct color when used in Windows 10

![image001](https://user-images.githubusercontent.com/33945730/154124719-c42cb492-bd16-4014-a2db-820d80686b68.png)

**Comment by Xtopher98** [comment]
I have forked the repo and have a fix implemented. Will be creating a PR soon

**Comment by Anselmoo** [comment]
@Xtopher98 thx, we are looking forward

**Comment by Anselmoo** [comment]
@Xtopher98 Thank you very much for the PR!

It looks like that the broken color scheme comes from the new `gitkraken` release **8.3.0** because the old scheme is also broken under MAC.

<img width="491" alt="image" src="https://user-images.githubusercontent.com/13209783/154133565-3461f1f9-fb83-46b5-b45c-05f06eb70dd5.png">


However I am not sure if the main branch color is right one,

<img width="490" alt="image" src="https://user-images.githubusercontent.com/13209783/154133474-a086687a-d356-46fe-8ad1-441db1344792.png">


because it very darken and make it hard to follow the main branch.

**Comment by Anselmoo** [comment]
Please also update the README.md image because it will not be up to date anymore.

**Comment by Xtopher98** [comment]
@Anselmoo Thanks for the feedback! Personally I like the darker master branch but I agree it may not be the best overall. I'll get that updated and update the readme as well and create a new PR.

**Comment by Anselmoo** [comment]
@Xtopher98 thx for the quick reply, this fix is really mandatory for all platforms!

**Comment by Xtopher98** [comment]
@Anselmoo Changes has been made and pushed to the same PR #8 as before.
